cmd/kind/version/version.go

```diff
@@ -23,19 +23,41 @@ import (
 	"github.com/spf13/cobra"
 )
 
-// Version is the kind CLI version
-const Version = "v0.6.0-alpha"
+// Version returns the kind CLI Semantic Version
+func Version() string {
+	v := VersionCore
+	// add pre-release version info if we have it
+	if VersionPreRelease != "" {
+		v += "-" + VersionPreRelease
+		// if commit was set, add the + <build>
+		// we only do this for pre-release versions
+		if GitCommit != "" {
```

Comment:
to consider in a follow-up: we might want to instead (or additionally) have the release scripts to `export COMMIT=""` to make it easier to reproducibly build the binaries from a source tarball. or instructions for doing `COMMIT=<release-commit> make build`

Reply:
I think it is probably better to include the commit, and then if someone wants to reproduce the binaries without git they can set `COMMIT=` (the makefile should already support this)